test/WebSites/ModelBindingWebSite/FromNonExistantBinderAttribute.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace ModelBindingWebSite
 {
-    public class FromNonExistantBinderAttribute : Attribute, IBinderMetadata
+    public class FromNonExistantBinderAttribute : Attribute
     {
     }
```

Comment:
Does this still have a point? this likely needs to change to have the desired impact